### ISSUE TITLE
Map container to external ports instead of ephemeral

### ISSUE THREAD

**Issue by patrickdaj** [ovrclk/akash#1276]
**Is your feature request related to a problem? Please describe.**
No ability to map container to external ports 1-to-1 other than port 80.

**Describe the solution you'd like**
Create a method to map container to external ports 1-to-1 rather than using ephemeral.

**Describe alternatives you've considered**
None.

**Additional context**
None

**Comment by hydrogen18** [comment]
@patrickdaj I don't think this is something we can do at this time. In the future Akash would like to support this, but I can't provide a time table of features like this.
